vendor.conf

```diff
@@ -32,17 +32,17 @@ github.com/hashicorp/golang-lru                     v0.5.3
 github.com/imdario/mergo                            v0.3.7
 github.com/konsorten/go-windows-terminal-sequences  v1.0.3
 github.com/matttproud/golang_protobuf_extensions    v1.0.1
-github.com/Microsoft/go-winio                       v0.4.14
+github.com/Microsoft/go-winio                       fc70bd9a86b5562b3b5eb1040e803febee1e90a1
 github.com/Microsoft/hcsshim                        v0.8.9
 github.com/opencontainers/go-digest                 v1.0.0
 github.com/opencontainers/image-spec                v1.0.1
 github.com/opencontainers/runc                      v1.0.0-rc10
 github.com/opencontainers/runtime-spec              v1.0.2
 github.com/pkg/errors                               v0.9.1
-github.com/prometheus/client_golang                 v1.3.0
-github.com/prometheus/client_model                  v0.1.0
-github.com/prometheus/common                        v0.7.0
-github.com/prometheus/procfs                        v0.0.8
+github.com/prometheus/client_golang                 v1.6.0
+github.com/prometheus/client_model                  v0.2.0
+github.com/prometheus/common                        v0.9.1
+github.com/prometheus/procfs                        v0.0.11
```

Comment:
Wondering if we need to move these to a new section of they're both a dependency for containerd and for k8s.

(Guess not the biggest issue if we would go ahead and merge this repository into the main containerd repo)

Reply:
+1 to leave it alone as we should merge this repo into containerd/containerd.